test/unit/redact-secrets.test.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * (C) Copyright IBM Corp. 2024.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { redactSecrets } = require('../../dist/lib/private-helpers');
+
+describe('redactSecrets()', () => {
+  it('misc. tests', async () => {
+    const inputStrings = [
+      'Authorization: Bearer secret',
+      'Authorization: secret\nX-Author: secret',
+      'apikey=81KiI5Zm2kjOWnrSglhtnDJn3u0kfv&grant_type=apikey&response_type=cloud_iam',
```

Comment:
I'm guessing detect secrets will have a field day with this 😂 and it looks like you haven't updated the baseline yet - just a reminder

Reply:
good catch!